bundles/org.openhab.binding.ecobee/src/main/java/org/openhab/binding/ecobee/internal/handler/EcobeeThermostatBridgeHandler.java

```diff
@@ -179,8 +163,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         });
     }
 
-    public void setDiscoveryService(SensorDiscoveryService discoveryService) {
-        this.discoveryService = discoveryService;
+    public boolean isBackgroundDiscoveryEnabled() {
+        return ((EcobeeAccountBridgeHandler) getBridge().getHandler()).isBackgroundDiscoveryEnabled();
     }
```

Comment:
Is this method ever called?

Reply:
It was before I consolidated the two discovery services into one. It can be removed now. Good spot.